src/components/questions/QuestionsAnswers.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import axios from 'axios'
+class QuestionsAnswers extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      productId: null,
+      questionsList: [],
+      term: ''
+    }
+  }
+
+  componentDidMount() {
+
+    const { productId } = this.props;
+
+    let questionConfig = {
+      method: 'get',
+      url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-nyc/qa/questions?product_id=${productId}`,
+      headers: {
+        'Authorization': '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'
+      }
+    };
+
+    if (productId) {
+      axios(questionConfig)
+        .then((res) => {
+          this.setState({
+            productId: productId,
```

Comment:
Why does productId need to be in state?

Reply:
> 

Definately not intentional. I think this is just because I added the directory sorry!

Reply:
Would you mind refactoring? It might get confusing later on.

Reply:
I refactored, but I was wondering if you could elaborate on what is confusing about it? I have to make post request with the productId and figured having it in the state would be useful instead of having to prop drill it 4-5 components deep.

Reply:
Wouldn't you still have to prop drill? Since state only applies to the current component, not its children?